src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetRandomCommand.cs

```diff
@@ -396,11 +396,14 @@ protected override void BeginProcessing()
                         this.ThrowMinGreaterThanOrEqualMax(min, max);
                     }
 
-                    int randomNumber = this.Generator.Next(min, max);
-                    Debug.Assert(min <= randomNumber, "lower bound <= random number");
-                    Debug.Assert(randomNumber < max, "random number < upper bound");
+                    for (int i = 0; i < Math.Max(1, this.Count); i++)
+                    {
+                        int randomNumber = this.Generator.Next(min, max);
```

Comment:
I don't think `this.` is usually needed here. Is there a naming conflict that requires this? If not, we don't generally use it where it's not explicitly required. 🙂 

There are a couple of these, just tagging this one.

Reply:
Done. I've removed all `this.` invocations from this commandlet. :)

Reply:
Cheers! 😄